services/elasticloadbalancing/src/main/resources/codegen-resources/customization.config

```diff
@@ -138,7 +137,7 @@
         "TooManyAccessPointsException" : "TooManyLoadBalancersException",
         "InvalidEndPointException" : "InvalidInstanceException"
     },
-    "sendEmptyAutoConstructedListAsEmptyList" : {
+    "sendNullAsEmptyListMembers" : {
```

Comment:
Sounds good to me! I like the idea that customers have to explicitly set an empty list in order to wipe out the policies instead of that being the default if they don't set it.

Do we know why this was put in to begin with? This feature was added just for ELB, so someone had a reason to do it.